src/main/java/pl/multiplex/multiplexreservation/dao/MovieRepo.java

```diff
@@ -0,0 +1,10 @@
+package pl.multiplex.multiplexreservation.dao;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import pl.multiplex.multiplexreservation.dao.entity.Movie;
+
+
+@Repository
+public interface MovieRepo extends CrudRepository<Movie, Long> {
```

Comment:
W innych przypadkach używasz PagingAndSorting, a tu Crud, czemu?

Reply:
W pierwszej wersji wszędzie był Crud, potem pozamieniałem na PagingAndSorting i najwyraźniej przegapiłem to jedno miejsce.